markdown/en/ff-asn.md

```diff
@@ -25,6 +25,32 @@ If you set the ``[filter-style]` attribute properly the `ff-asn-group` acts as d
 </ff-asn>
 ```
 
```

Comment:
maybe also ninja-fix the styling mistake in line 17?

Reply:
- I think we should be more precise and state, that u can use the native select for the "hiddenLinks"
- Not sure, but I would preffer the special case of the select box not quite at the beginning of the docu